canonical/ubuntu.com
# Build /ai/mlops

## Done

- Built new page as per [doc](https://docs.google.com/document/d/18AMX_-3zjBNUNGr0X5QtjKhsUzAc2k8VM1-opZEGh4k/edit) and [design](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248%3A1446&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13146.demos.haus/ai/mlops
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5512
